test-helpers/Cargo.toml

```diff
@@ -39,3 +39,4 @@ http = "1.1.0"
 rustls = { version = "0.21.2" ,features = ["dangerous_configuration"] }
 rustls-pemfile = "1.0.2"
 tokio-tungstenite = { git = "https://github.com/shotover/tokio-tungstenite", features = ["rustls-tls-native-roots"], rev = "2f1cc11e491c2d0401d01a83f623623182784b3a" }
+pretty_assertions = "1.4.0"
```

Comment:
may as well make this a dev-dependency

Reply:
It's in the "test-helpers" module which means it's already only included as dev-dependency.